docs/Apps/Metabolomic Data/Polly-PeakML.md

```diff
@@ -0,0 +1,275 @@
+
+    Slider - Using Slider, the user can set the range of the classified label to be bad or signal accordingly. For example, if the range is set to 0.0 to 0.3 (that is marked by red) this determines that the group has a probability of less than 0.3 would be noise. Similarly, the peak-groups with a probability of more than 0.6 (marked by green) would be a signal ( that is a good peak-group).
+
+    The gray region in slider from 0.3 to 0.6 is marked as - ‘Maybe good peak-group’
+
+    ![Slider](../../img/Polly-PeakML/slider.png) <center>**Figure 8.** Adjust slider</center>
+
+* Caveats 
+    
+    * The ML model is for feature detection. it can distinguish between good and noise peak-groups. It's not trained to distinguish between matched and unmatched peak-groups for targeted analysis. 
+
+    * For targeted analysis, it can eliminate noise peak-groups but required manual curation to select the matched peak-group from all the good peaks that are detected for a given m/z and RT.
+
+# Tutorial
+
+**Classified Peak Table**
+
+* Peak-tables have two additional columns than usual, namely, *‘Labels’* and *‘Classification probability’*. 
+
+* Labels - Polly-peakML classifies the peak-groups into six different classes.
+    
+    * Signal: Good peak-group.
+    * Signals with correlation: Good peak-group correlated with other peaks within the same RT window.
+    * Signal with cohort-variance: Good peak-group showing some interesting patterns across defined cohorts.
+    * Signals with both correlation and cohort-variance: Good peak-group with both the above-mentioned properties.
+    * Noise: Bad peak-group.
+    * MayBe Good: Peak-group may be a real signal or not.
+
+* Classification Probability - The probability value which we can assure how much model is certain that it is noise or signals.
+
+![Classified Peak Table](../../img/Polly-PeakML/classifiedPeakTable.png) <center>**Figure 9.** Classified Peak Table</center>
+
+**Labels drop-down in the peak-table menu** 
+
+Using ‘labels’ dropdown, a user can filter out the classes that he doesn’t want to appear in the table. 
+
+![Labels](../../img/Polly-PeakML/labelsDropdown.png) <center>**Figure 10.** Understanding Labels</center>
+
+*  Percentage beside the *Label* name depicts the percentage of the peak-groups that are present in the current peak-table. For example, ‘Noise(13.74%)’ represents that 13.74% of all peak-groups are noise i.e bad peak-groups. 
+
+* The user would need to uncheck the label that he wants to filter out in the peak-table.
+
+![Filter Peak-groups](../../img/Polly-PeakML/filterPeakgroups.png) <center>**Figure 11.** Filter peak groups using *Labels* dropdown</center>
+
+**Relabel peak-groups**
+
+There may be the chance that the user used the slider parameters in the peak-detection dialog to be stringent. Using, Re-label option the user can change the limit of noise and signal and the peak-table would be re-classified based on their probability score. 
```

Comment:
If in case you have applied stringent slider parameters in the *peak detection* dialog. 
Relabel in italics
*you can